### PR TITLE
Add log double grad

### PR DESCRIPTION
**Pull request by joey12300**
### PR types

New features
### PR changes

OPs
### Describe

Add double gradients of log

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_activation_nn_grad.py [hunk 1]
```diff
@@ -147,5 +148,29 @@ def test_grad(self):
...
+class TestLogDoubleGradCheck(unittest.TestCase):
+    @prog_scope()
+    def func(self, place):
+        shape = [2, 3, 7, 9]
+        eps = 1e-6
+        dtype = np.float64
+
+        x = layers.data('x', shape, False, dtype)
+        x.persistable = True
+        y = paddle.log(x)
```
Comment (ZHUI) [inline python/paddle/fluid/tests/unittests/test_activation_nn_grad.py]: 这个要check pick到1.8， paddle.log对应的代码得修改

Comment (joey12300) [reply python/paddle/fluid/tests/unittests/test_activation_nn_grad.py]: fixed

**Review by wawltor — DISMISSED** [review]
LGTM

**Review by wawltor — APPROVED** [review]
LGTM

**Pull request merged**